app/src/main/java/unipd/se18/ocrcamera/CameraActivity.java

```diff
@@ -0,0 +1,123 @@
+/*
+ *  The main goal for this project is to test older Camera API (15 to 19) to see if it's
+ *  plausible to support these devices and lower the minimum API requirement on CommonDemo
+ *  so we can support full range of Android devices (API 15 and higher is 100% demographic)
```

Comment:
Yes. When creating a new project in Android Studio you have to set the minimum API requirement for your app; with 15 and higher you are targeting all (it's actually 99.8%, but Android Studio approximates to 100%  and I don't really get why since out of more or less 2 billion Android devices, **2% is still 40000000 devices with API lower than 15**) Android devices.